pkg/trace/api/profiles.go

```diff
@@ -68,7 +71,17 @@ func (r *HTTPReceiver) profileProxyHandler() http.Handler {
 	if err != nil {
 		return errorHandler(err)
 	}
-	tags := fmt.Sprintf("host:%s,default_env:%s", r.conf.Hostname, r.conf.DefaultEnv)
+	tags := fmt.Sprintf("host:%s,default_env:%s,agent_version:%s", r.conf.Hostname, r.conf.DefaultEnv, info.Version)
+	if r.conf.IsFargate {
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
+		orch := fargate.GetOrchestrator(context.Background())
+		cancel()
+		if err := ctx.Err(); err != nil && err != context.Canceled {
+			log.Warnf("Failed to detect if running in AWS Fargate. If you are in a Fargate instance, this may cause issues with your profiles: %v", err)
```

Comment:
```suggestion
			log.Warnf("Failed to get Fargate orchestrator. This may cause issues with your profiles: %v", err)
```
This isn't failed detection, this is failure to get the orchestrator. Furthermore, we already know we are in a Fargate instance because we are in this clause to begin with.